### PR TITLE
Configures log parsing with logstash and filebeat

### PR DESCRIPTION
**Pull request by waldemarnt**
Configures Logstash to:
- listen Filebeat from port 5044.
- parse log content using json plugin.
- to send output data to Elasticsearch.

Configures Filebat to:
- check log files in /var/log directory every 10s.
- to send logs to Logstash.

Updates Docker Compose to orchestrate all services like ELK and Filebeat.

### CODE REVIEW

**Comment by andressa** [comment]
👍

**Comment by lucaslago** [comment]
👍

File: filebeat/filebeat.yml [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+  # Enable debug output for selected components. To enable all selectors use ["*"]
+  # Other available selectors are beat, publish, service
+  # Multiple selectors can be chained.
+  selectors: ["*"]
+
+  # Sets log level. The default log level is error.
+  # Available log levels are: critical, error, warning, info, debug
+  level: warning
+
+
```
Comment (glauco) [inline filebeat/filebeat.yml]: Given we already know how we want to use Filebeat, could we remove the configuration file comments?

Comment (waldemarnt) [inline filebeat/filebeat.yml]: Fixed https://github.com/agco-fuse/fuse-equipment-api/pull/57/commits/4d454ffec07d16eb08f57d50b3591a493357e862

File: filebeat/filebeat.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+############################# Filebeat ######################################
```
Comment (glauco) [inline filebeat/filebeat.yml]: Given we removed all comments, could we remove this one as well?

Comment (waldemarnt) [inline filebeat/filebeat.yml]: It's so beautiful. Was removed in this commit https://github.com/agco-fuse/fuse-equipment-api/pull/57/commits/68590fb107c602ea05b48d6337ed3bfe12fce7cb :)

**Comment by glauco** [comment]
🐑 🇮🇹

**Pull request merged**